virttest/vt_cluster/proxy.py

```
@@ -0,0 +1,138 @@
+                    specific_exception_class = getattr(actual_root_mod, exc_type_name)
+                else:
+                    # Try to find a built-in or globally available exception
+                    specific_exception_class = getattr(
+                        importlib.import_module("builtins"), exc_type_name, None
+                    )
+                    if specific_exception_class is None:
+                        # Fallback or search in other common modules if necessary
+                        specific_exception_class = eval(exc_type_name)
+
+                if isinstance(kargs, dict):
+                    raise specific_exception_class(**kargs)
```

Ruff (0.12.2)

Abstract `raise` to an inner function

(TRY301)